### ISSUE TITLE
pip dependency conflicts with docutils

### ISSUE THREAD

**Issue by smojef** [zauberzeug/nicegui#399]
`pip` just gave me this error message while updating another package:
```
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
nicegui 1.1.8 requires docutils<0.18.0,>=0.17.1, but you have docutils 0.19 which is incompatible.
```
I managed to downgrade docutils as a workaround.
Should nicegui be compatible with the latest version?

**Comment by rodja** [comment]
Thanks for pointing this out @smojef. Indeed the version of docutils is outdated. But I realized that docutils is only used on the website: 

https://github.com/zauberzeug/nicegui/blob/57d6ad884457cc78e58d1457f3a5a6dad343a654/website/example.py#L46

Therefore I'll move it to the dev dependencies.

**Comment by rodja** [comment]
Just released v1.1.9
